### PR TITLE
chore(deps): update hyper v1.0

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

**Review by tadeohepperle — APPROVED** [review]
Nice!

File: examples/hyper_server.rs [hunk 1]
```diff
@@ -21,33 +21,64 @@
...
+		"Listening on http://{:?} — connect and I'll echo back anything you send!",
+		listener.local_addr().unwrap()
+	);
 
-	println!("Listening on http://{} — connect and I'll echo back anything you send!", server.local_addr());
-	server.await?;
+	loop {
+		let stream = match listener.accept().await {
+			Ok((stream, addr)) => {
+				log::info!("Accepting new connection failed: {addr}");
```
Comment (niklasad1) [inline examples/hyper_server.rs]:
```suggestion
				log::info!("Accepting new connection: {addr}");
```

**Review by jsdw — APPROVED** [review]
lgtm assuming the example works :)

**Comment by niklasad1** [comment]
yupp :)

**Pull request merged**